sdk/core/azure-core/azure/core/tracing/_abstract_span.py

```diff
@@ -7,7 +7,7 @@
 from enum import Enum
 from urllib.parse import urlparse
 
-from typing import Any, Sequence, Optional, Union, Callable, Dict, Type
+from typing import Any, Sequence, Optional, Union, Callable, Dict, Type, runtime_checkable, Generic, TypeVar
```

Comment:
We import runtime_checkable, but don't use it.

Reply:
Yes, CI caught it :). It was meant for the `AbstractSpan`